auto_causality/params.py

```diff
@@ -468,14 +468,12 @@ def _configs(self):
                     "model_t_xw": propensity_model,
                     "model_t_xwz": deepcopy(propensity_model),
                     "model_final": final_model,
-                    "discrete_treatment": False,
                 },
                 search_space={
                     "fit_cate_intercept": tune.choice([0, 1]),
                     "mc_agg": tune.choice(["mean", "median"]),
                 },
                 defaults={
-                    "fit_cate_intercept": True,
```

Comment:
Why did you remove this? If you don't specify it explicitly, FLAML might sample it from the search space instead. Therefore it's best to match the serarch_space and defaults fields one for one